### DISCUSSION TITLE
Project Ideation

### DISCUSSION THREAD

**Discussion by ahmad-reza619** [KongkowITPekanbaru/masjid-network#2] [Ideas]
Hello! 👋 

Disini kita bakal bahas tentang ideation sistem yang akan kita buat, silahkan comment atau share idenya masing masing ya tentang bagaimana flownya atau fitur fitur nya apa aja

**Comment by Eore** [comment]
Dri aku target awalnya pencatatan kas masjid, biasanya ada pake buku, atau papan tulis, klo ad sistem nya, history kas bisa disajikan dlm bntuk report yg mudah dimengerti umat (pake grafik ato semacamnya), selain itu dri umat sndiri bisa akses lngsung, jdi lebih transparan

**Reply by Eore** [reply]
Untuk jenis kasnya, seperti yg di bahas sebelumnya, bisa pake tag aja, itu simple nya ya

**Reply by jufianto** [reply]
wkwkw rupanya udah ada, pindah ke discussion sebelah aja ndak? karna udah ke banyak diskusi yang disana juga

**Reply by Eore** [reply]
Okeee @jufianto udh diinit dsitu jg kan

**Comment by jufianto** [comment]
oke di lock aja ini ya
